crates/client/rpc-core/src/lib.rs

```diff
@@ -35,13 +35,13 @@ pub struct FunctionCall {
 }
 
 /// A block hash, number or tag
-/// TODO: fix block_tag
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 pub enum BlockId {
     #[serde(rename = "block_hash")]
     BlockHash(BlockHash),
     #[serde(rename = "block_number")]
     BlockNumber(BlockNumber),
+    #[serde(rename = "block_tag")]
```

Comment:
should rather be `#[serde(skip)]` no ? since it's not specified as `{ block_tag: 'latest'}` but directly `'latest'`

Reply:
Yep I think you are right. As weird as it make the API, that's how it's defined